app/models/work_package.rb

```diff
@@ -757,6 +757,32 @@ def editable_by?(user)
     allowed
   end
 
+  def inherit_done_ratio_from_leaves
```

Comment:
Assignment Branch Condition size for inherit_done_ratio_from_leaves is too high. [23.35/15]